dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,123 @@
+
+CWPKI2019I=CWPKI2019I: The account URL provided by the ACME certificate authority at the {0} URI is {1}.
+CWPKI2019I.explanation=The account was successfully created.
+CWPKI2019I.useraction=No action is required.
+
+CWPKI2020E=CWPKI2020E: The ACME service could not read the domain key file for the ACME certificate authority account at the {0} URI. The file location is {1}. The error is {2}.
+CWPKI2020E.explanation=The domain key file for the certificate authority account could not be opened. This can occur if file permissions are incorrect or the file does not exist.
+CWPKI2020E.useraction=Review the error message for details on the failure. Verify that the file location is correct and has the correct file permissions.
+
+CWPKI2021E=CWPKI2021E: The ACME service could not read the account key file for the ACME certificate authority account at the {0} URI. The file location is {1}. The error is {2}.
+CWPKI2021E.explanation=The account key file for the certificate authority account could not be opened. This can occur if file permissions are incorrect or the file does not exist.
+CWPKI2021E.useraction=Review the error message for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+CWPKI2022E=CWPKI2022E: The ACME service could not write to the domain key file for the ACME certificate authority account at the {0} URI. The file location is {1}. The error is {2}.
+CWPKI2022E.explanation=Could not write to the domain key file for the certificate authority account. This can occur if file permissions are incorrect or the file does not exist.
+CWPKI2022E.useraction=Review the error message for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+CWPKI2023E=CWPKI2023E: Failed to write the account key file for the ACME certificate authority account at the {0} URI. The file location is {1}. The error is {2}.
+CWPKI2023E.explanation=Could not write to the account key file for the certificate authority account. This can occur if file permissions are incorrect or the file does not exist.
+CWPKI2023E.useraction=Review the error message for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+CWPKI2024E=CWPKI2024E: The ACME service failed to revoke the requested certificate for the ACME certificate authority at the {0} URI. The certificate is serial number {1}. The error is {2}.
+CWPKI2024E.explanation=A certificate revoke request failed. The certificate might not be revoked and could still be in use.
+CWPKI2024E.useraction=Review the error message for details on the failure.
+
+CWPKI2025W=CWPKI2025W: The ACME service received a certificate revoke request, but the account key pair for the ACME certificate authority at the {0} URI could not be loaded.
+CWPKI2025W.explanation=Could not revoke a certificate because the account key pair could not be loaded.
```

Comment:
**change**
Could not revoke a certificate because the account key pair could not be loaded.
 **to**
The certificate revoke request failed because the account key pair could not be loaded.

if thats accurate...